tutorials/37-offline.md

```diff
@@ -0,0 +1,64 @@
+Enketo may be [configured](./tutorial-10-configure.html#offline-enabled) to support offline access to forms. When this configuration is set to `true`, accessing a form in offline-capable mode _while online_ will allow that form to be available for subsequent requests _while offline_.
+
+**Note:** unless specified otherwise, all of the caching and storage discussed below refers to storage _on an individual user's device and browser_. Also unless specified otherwise, caching is not used when the Enketo deployment is not configured to support offline forms, or when the user accesses a form in online-only mode (which is the default).
+
+### Offline-capable mode
+
+#### Service Worker and Cache Storage
+
+The Enketo app registers a [Service Worker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API), which intercepts some network requests in order to [cache](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage) certain resources as they're requested. The following resources are stored in one of two caches:
+
```

Comment:
Should the section header go here? The two caches are Cache Storage and IndexedDB, right? The current structure makes it seem like the two caches are "Common static assets" and "Resources associated"

Reply:
I got all twisted here! Yes, there are two separate caches in Cache Storage (https://github.com/enketo/enketo-express/pull/465/files#diff-e6ed1633d2dc274af2512a39c50c01f8a30752d45efa2a134a7acc5494ee2012R1)